utils/test/patterns_tests/ClusterAnalysisTestCase.py

```diff
@@ -39,54 +42,46 @@
 class ClusterAnalysisTestCase(unittest.TestCase):
     ''' Test all tools and methods related to the Cluster Analysis tool
     in the Incident Analysis toolbox'''
-    
     inputPointsFeatures = None
-    
+
     def setUp(self):
         if Configuration.DEBUG == True: print("     ClusterAnalysisTestCase.setUp")
         UnitTestUtilities.checkArcPy()
-        
+
         Configuration.incidentDataPath = DataDownload.runDataDownload(Configuration.patternsPaths, Configuration.incidentGDBName, Configuration.incidentURL)
         if (Configuration.incidentScratchGDB == None) or (not arcpy.Exists(Configuration.incidentScratchGDB)):
             Configuration.incidentScratchGDB = UnitTestUtilities.createScratch(Configuration.incidentDataPath)
         Configuration.incidentInputGDB = os.path.join(Configuration.incidentDataPath, Configuration.incidentGDBName)    
-        
+
         UnitTestUtilities.checkFilePaths([Configuration.incidentDataPath, Configuration.incidentInputGDB, Configuration.patterns_ProToolboxPath, Configuration.patterns_DesktopToolboxPath])
-      
+
         # set up inputs
         self.inputPointsFeatures = os.path.join(Configuration.incidentInputGDB, "Incidents")
-        
+
     def tearDown(self):
         if Configuration.DEBUG == True: print("     ClusterAnalysisTestCase.tearDown")
         UnitTestUtilities.deleteScratch(Configuration.incidentScratchGDB)
-        
-    def test_cluster_analysis_pro(self):
-        arcpy.AddMessage("Testing Cluster Analysis (Pro).")
-        self.test_cluster_analysis(Configuration.patterns_ProToolboxPath)
-    
-    def test_cluster_analysis_desktop(self):
-        arcpy.AddMessage("Testing Cluster Analysis (Desktop).")
-        self.test_cluster_analysis(Configuration.patterns_DesktopToolboxPath)
-        
-    def test_cluster_analysis(self, toolboxPath):
-        try:
-            if Configuration.DEBUG == True: print("     ClusterAnalysisTestCase.test_cluster_analysis")
 
-            arcpy.ImportToolbox(toolboxPath, "iaTools")
-            outputClusterFeatures = os.path.join(Configuration.incidentScratchGDB, "outputClusters")
+    def test_cluster_analysis_pro(self):
```

Comment:
Before continuing with splitting up the test cases into "_pro" and "_desktop" test case versions - we should take a closer look at the source of the original problem capturing/trapping exceptions/errors